apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -703,12 +752,12 @@ contract DisputableVoting is DisputableAragonApp, IForwarder {
     }
 
     /**
-    * @dev Internal function to check if a voter has already voted on a certain vote
+    * @dev Internal function to check if the vote for a voter has already been casted
     *      It assumes the pointer to the vote is valid
-    * @return True if the given voter has not voted on the requested vote
+    * @return True if a vote for the given voter has already been casted
     */
-    function _hasNotVotedYet(Vote storage vote_, address _voter) internal view returns (bool) {
-        return _voteCaster(vote_, _voter) != _voter;
+    function _hasCastedVote(Vote storage vote_, address _voter) internal view returns (bool) {
+        return _voterState(vote_, _voter) != VoterState.Absent;
```

Comment:
This changes also means that representatives can’t change votes either, which is consistent with holders not being able.

Reply:
Exactly, since multiple extensions are allowed, if we allow representatives to change their vote, then they could extend votes indefinitely due to quiet ending